peribolos/knative-sandbox.yaml

```diff
@@ -155,7 +155,6 @@ orgs:
     - matzew
     - mayaspivak
     - maximilien
-    - mdemirhan
```

Comment:
Is this by accident or intentional? (as it does not seem to be part of the original intent of the PR)

Reply:
It turns out that Mustafa and Tomas haven't accepted invites to the knative-sandbox org, so aren't valid members in an OWNERS_ALIASES file, which Prow checks.

Reply:
(Also, Mustafa has switched teams/companies and headed back to Microsoft; Tomas accepted a role in a different part of Google.)